### ISSUE TITLE
'Reset' classifications and count

### ISSUE THREAD

**Issue by chrislintott** [zooniverse/Panoptes-Front-End#273]
Project owners and collaborators should be able to 'reset' their classifications without changing a workflow.

Use case : I have created a project and shared with my collaborators, who have noodled around on it. I now want to share it with those who are actually going to do the classifications. I will want the classification count to be reset and probably a clean set of classifications.

**Comment by aliburchard** [comment]
This should address @edpaget 's question about when we start counting towards retirement. We are always "counting" towards retirement, but the 'reset' button then allows you to start the count over so that everything you've done while noodling around doesn't mess with your data.

**Comment by edpaget** [comment]
Why not just not count towards retirement until a project is in a 'launched' or 'live' mode?

**Comment by camallen** [comment]
I'm with @edpaget on this one.

**Comment by aliburchard** [comment]
What about projects that don't launch through zooniverse? Based on the stages @chrislintott defined in #240, there is no "live" stage for non zooniverse-approved projects.

**Comment by camallen** [comment]
There is the `lock` stage that can be used to count towards retirement, i.e. only count if lock = T.

**Comment by edpaget** [comment]
@aliburchard I see 'Launch' and 'Launch-Approved' states I don't see why Launch with or without approval couldn't start the counts.

**Comment by edpaget** [comment]
Nevermind I see that's supposed to be a request for review. Anyway I think @camallen has it right. Locking your project starts counts unlocking resets them.

**Comment by aliburchard** [comment]
Okay, so we're really treating "lock" == "live" (which I'm fine with, and think it's a matter of presenting it to the project builders with the right text on the front end). But one question: are we planning that project owners and collaborators can make changes to project *text* while a project is "locked"? (And, as brought up in another issue, as text changes are minor version changes, they can be lumped together for aggregating.)

**Comment by chrissnyder** [comment]
No idea where this one is. I think it was also taken care of with the project visibility work. @edpaget ?

**Comment by edpaget** [comment]
The reset hasn't been implemented (as far as I know), but things are only counted once the project is in the 'live' state.

**Comment by mschwamb** [comment]
I'm currently working on parsing the comet hunters and planet four classifications. As we did make a bunch of changes during development, being able to not have the classifications from the beta test and before launch in the same csv would make things easier. I sorted it with version number for the workflow, but ti would be handy if there could be one clearing of the database or only reporting of classifications from when the site was live in the data dumps.  I also presume @ggdhines  the current aggregation code  doesn't yet filter out marks from before launch and after launch?
